balancer/weightedroundrobin/balancer.go

```diff
@@ -404,8 +464,13 @@ func (p *picker) Pick(info balancer.PickInfo) (balancer.PickResult, error) {
 // When needed, it also tracks connectivity state, listens for metrics updates
 // by implementing the orca.OOBListener interface and manages that listener.
 type weightedSubConn struct {
+	// These fields are set at construction time and read only after, so no need
+	// for synchronization.
```

Comment:
`// The following fields are immutable.`?

Reply:
Sure this is how I see it documented throughout the codebase in other places but it's equivalent to yours so switched.